dotcom-rendering/src/web/layouts/headers/ImmersiveHeader.tsx

```diff
@@ -263,6 +263,10 @@ export const ImmersiveHeader = ({ CAPI, NAV, format }: Props): JSX.Element => {
 										webPublicationDateDeprecated={
 											CAPI.webPublicationDateDeprecated
 										}
+										hasStarRating={
```

Comment:
This extra prop is required to allow us to decide the bottom padding of the ArticleHeadline (?)

Reply:
Yep. When there are stars present it changes the amount of padding needed because the presence of the stars has already pushed the content down. See below

<img width="677" alt="Screenshot 2022-03-30 at 14 58 29" src="https://user-images.githubusercontent.com/1336821/160839871-0b3aed2c-0f97-48f2-a616-60b202cde7e9.png">